sa/sa.go

```diff
@@ -564,17 +564,17 @@ func (ssa *SQLStorageAuthority) CountOrders(ctx context.Context, req *sapb.Count
 	return &sapb.Count{Count: count}, nil
 }
 
-// hashNames returns a hash of the names requested. This is intended for use
+// HashNames returns a hash of the names requested. This is intended for use
 // when interacting with the orderFqdnSets table.
-func hashNames(names []string) []byte {
+func HashNames(names []string) []byte {
```

Comment:
This is a pretty straight forward function, but as mentioned on another review comment we should be testing it, especially with multiple values in the string.